### PR TITLE
Add widget

### PR DESCRIPTION
**Pull request by smartman**

### CODE REVIEW

File: resources/views/login-widget.blade.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            crossorigin="anonymous"
+    ></script>
+    <script>
+      const widgetHolder = document.getElementById('widgetHolder');
+      const eidEasyWidget = document.createElement('eideasy-widget');
+      const settings = {
+        countryCode: 'EE', // ISO 3166  two letter country code
+        language: 'et', // ISO 639-1 two letter language code,
+        sandbox: true,
+        clientId: '2IaeiZXbcKzlP1KvjZH9ghty2IJKM8Lg',
```
Comment (smartman) [inline resources/views/login-widget.blade.php:34]: It should use process.env.MIX_EID_CLIENTID

File: resources/views/login-widget.blade.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            Login Widget
+        </h1>
+        <div style="margin-top: 20px; margin-bottom: 10px;">
+            <label for="langPicker">Language:</label>
+            <select name="langPicker" id="langPicker">
+                <option value="en">en</option>
+                <option value="et" selected>et</option>
+                <option value="lv">lv</option>
+                <option value="lt">lt</option>
+                <option value="ru">ru</option>
```
Comment (smartman) [inline resources/views/login-widget.blade.php:15]: Lets use full English language names

File: resources/views/login-widget.blade.php (hunk 1 above)
Comment (matsjoonas) [reply resources/views/login-widget.blade.php]: we now use the value from the .env for the clientId.

File: resources/views/login-widget.blade.php (hunk 2 above)
Comment (matsjoonas) [reply resources/views/login-widget.blade.php]: Added full english names.

**Pull request merged**